### ISSUE TITLE
Use of SND-MSG and variable Msg causes linter issues.

### ISSUE THREAD

**Issue by m-tyler** [codefori/vscode-rpgle#309]
Use of SND-MSG and variable Msg causes linter issues.

Is this a simple fix? 

![image](https://github.com/codefori/vscode-rpgle/assets/1353665/18e19c31-f119-41df-a84f-dc6a5f59f1ce)

```
**FREE
  Dcl-S Msg Varchar(64);
  Msg = 'My message to the joblog';
  SND-MSG *INFO %MSG( 'MSG9997' : 'WFIMSGF' :Msg ) %TARGET( *SELF );
  *INLR = *ON;
  Return;
```

**Comment by worksofliam** [comment]
I will look into it. Thanks for your sample.

**Comment by worksofliam** [comment]
Fixed in 0.26.4.
